code/__DEFINES/access.dm

```diff
@@ -150,3 +150,164 @@
+#define COMMON_ACCESS 			list(ACCESS_MECH_MINING, ACCESS_MECH_MEDICAL, ACCESS_MECH_SECURITY, ACCESS_MECH_SCIENCE, \
+								ACCESS_MECH_ENGINE, ACCESS_AUX_BASE, ACCESS_PSYCHOLOGY, ACCESS_PHARMACY, ACCESS_NETWORK, \
+								ACCESS_WEAPONS, ACCESS_MINERAL_STOREROOM, ACCESS_SEC_DOORS, ACCESS_XENOBIOLOGY, \
+								ACCESS_MINING_STATION, ACCESS_MAILSORTING, ACCESS_MINING, ACCESS_RESEARCH, ACCESS_THEATRE, \
+								ACCESS_SURGERY, ACCESS_COURT, ACCESS_QM, ACCESS_VIROLOGY, ACCESS_LAWYER, ACCESS_LIBRARY, \
+								ACCESS_HYDROPONICS, ACCESS_CHEMISTRY, ACCESS_CONSTRUCTION, ACCESS_CARGO, ACCESS_ROBOTICS, \
+								ACCESS_KITCHEN, ACCESS_CREMATORIUM, ACCESS_JANITOR, ACCESS_BAR, ACCESS_CHAPEL_OFFICE, \
+								ACCESS_EXTERNAL_AIRLOCKS, ACCESS_MAINT_TUNNELS, ACCESS_ENGINE_EQUIP, ACCESS_ENGINE, \
+								ACCESS_GENETICS, ACCESS_RND, ACCESS_MORGUE, ACCESS_MEDICAL, ACCESS_FORENSICS_LOCKERS, \
+								ACCESS_BRIG, ACCESS_SECURITY)
+/// Command staff/secure accesses, think bridge/armoury, AI upload, notably access to modify ID cards themselves.
+#define COMMAND_ACCESS 			list(ACCESS_TOXINS_STORAGE, ACCESS_MINISAT, ACCESS_TCOMSAT, ACCESS_KEYCARD_AUTH, \
+								ACCESS_RC_ANNOUNCE, ACCESS_VAULT, ACCESS_ATMOSPHERICS, ACCESS_TECH_STORAGE, ACCESS_HEADS, \
+								ACCESS_TELEPORTER, ACCESS_ARMORY, ACCESS_AI_UPLOAD, ACCESS_CHANGE_IDS, ACCESS_TOXINS, \
+								ACCESS_EVA, ACCESS_GATEWAY, ACCESS_ALL_PERSONAL_LOCKERS)
+/// Private head of staff offices, usually only granted to most cards by trimming
+#define PRIVATE_COMMAND_ACCESS 	list(ACCESS_HOS, ACCESS_HOP, ACCESS_CE, ACCESS_CMO, ACCESS_RD)
+/// Captains private rooms.
+#define CAPTAIN_ACCESS 			list(ACCESS_CAPTAIN)
+/// Centcom area stuff
+#define CENTCOM_ACCESS 			list(ACCESS_CENT_BAR, ACCESS_CENT_CAPTAIN, ACCESS_CENT_TELEPORTER, ACCESS_CENT_STORAGE,\
+								ACCESS_CENT_LIVING, ACCESS_CENT_MEDICAL, ACCESS_CENT_SPECOPS, ACCESS_CENT_THUNDER, \
+								ACCESS_CENT_GENERAL)
+/// Syndicate areas off station
+#define SYNDICATE_ACCESS 		list(ACCESS_SYNDICATE_LEADER, ACCESS_SYNDICATE)
+/// Away missions/gateway/space ruins
+#define AWAY_ACCESS 			list(ACCESS_AWAY_GENERAL, ACCESS_AWAY_MAINT, ACCESS_AWAY_MED, ACCESS_AWAY_SEC, ACCESS_AWAY_ENGINE, \
+								ACCESS_AWAY_GENERIC1, ACCESS_AWAY_GENERIC2, ACCESS_AWAY_GENERIC3, ACCESS_AWAY_GENERIC4)
+/// Special
+#define CULT_ACCESS 			list(ACCESS_BLOODCULT)
+
+#define REGION_ACCESS_ALL_GLOBAL 		REGION_ACCESS_ALL_STATION + CENTCOM_ACCESS + SYNDICATE_ACCESS + AWAY_ACCESS + CULT_ACCESS
+#define REGION_ACCESS_ALL_STATION 		COMMON_ACCESS + COMMAND_ACCESS + PRIVATE_COMMAND_ACCESS + CAPTAIN_ACCESS
+#define REGION_ACCESS_GENERAL 			list(ACCESS_KITCHEN, ACCESS_BAR, ACCESS_HYDROPONICS, ACCESS_JANITOR, ACCESS_CHAPEL_OFFICE, ACCESS_CREMATORIUM, \
+										ACCESS_LIBRARY, ACCESS_THEATRE, ACCESS_LAWYER)
+#define REGION_ACCESS_SECURITY 			list(ACCESS_SEC_DOORS, ACCESS_WEAPONS, ACCESS_SECURITY, ACCESS_BRIG, ACCESS_ARMORY, ACCESS_FORENSICS_LOCKERS, \
```

Comment:
Probably not. I actually just wanted a load of stuff together in the same place during testing since I have a LOT of stuff to document.

One of my design goals was to actually move a lot of stuff out of globals procs and defines and into SSid_access, allowing all this stuff to be changed at runtime (events/traits/admemery/etc.)

Reply:
Following up, these are used in /datum/controller/subsystem/id_access/proc/setup_tgui_lists() to set up the accesses_by_region list in there. I'd keep these as defines to keep the subsystem code cleaner and instead I'll replace all code references to these defines with their subsystem counterparts.

Reply:
Aight